cowprotocol/accounting/fees/protocol_fees_by_order_class_3703312.sql

```diff
@@ -0,0 +1,44 @@
+with
```

Comment:
Can you follow the standard and include a short summary of what this query does as well as documentation for all required parameters.

Reply:
Oh right, forgot about it. Thanks!